operator/src/main/java/org/keycloak/operator/Config.java

```diff
@@ -19,6 +19,8 @@
 
 import io.smallrye.config.ConfigMapping;
 
+import java.util.Optional;
```

Comment:
[nitpick] leftover?

Reply:
:+1:

Reply:
@andreaTP Let's not nitpick in a prototype and already quite lengthy discussion. ;)